### PR TITLE
Create the layout for login page

### PR DESCRIPTION
**Pull request by letrungkien0210**
Hi @jakubknejzlik ,

Please review.

Thanks

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    <!--
+      This HTML file is a template.
+      If you open it directly in the browser, you will see an empty page.
+
+      You can add webfonts, meta tags, or analytics to this file.
+      The build step will place the bundled scripts into the <body> tag.
+
+      To begin the development, run `npm start` or `yarn start`.
+      To create a production bundle, use `npm run build` or `yarn build`.
+    -->
```
Comment (jakubknejzlik) [inline public/index.html]: Please remove unnecessary comments

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+.App-intro {
+  /*font-size: large;*/
+}
+
+@keyframes App-logo-spin {
+  from { transform: rotate(0deg); }
+  to { transform: rotate(360deg); }
+}
```
Comment (jakubknejzlik) [inline src/App.css:24]: Please remove unnecessary comments

File: src/App.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  render() {
+    return (
+      <div className="App">
+        {/*<ul>*/}
+          {/*<li><Link to='/'>Home</Link></li>*/}
+          {/*<li><Link to='/login'>Login</Link></li>*/}
+        {/*</ul>*/}
+        <Switch>
+          <Route path='/login' component={Login}/>
+        </Switch>
```
Comment (jakubknejzlik) [inline src/App.js]: Please remove commented list and change `/login` to `/`

File: src/index.css [hunk 4]
```diff
@@ -0,0 +1,5 @@
+body {
+  /*margin: 0;*/
+  /*padding: 0;*/
+  /*font-family: sans-serif;*/
+}
```
Comment (jakubknejzlik) [inline src/index.css]: Can we remove this file, if there's nothing used in it?

**Review by jakubknejzlik — CHANGES_REQUESTED** [review]

**Comment by letrungkien0210** [comment]
Hi @jakubknejzlik ,

Please review.

Thanks.

**Pull request merged**